.github/workflows/release_droid_release_on_maven_central.yml

```diff
@@ -23,8 +23,8 @@ jobs:
           gpg-private-key: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
           gpg-passphrase: MAVEN_GPG_PASSPHRASE
       - name: Publish to Central Repository
-        run: mvn clean --batch-mode -Dgpg.skip=false -DskipTests deploy
+        run: mvn --batch-mode -Dgpg.skip=false -DskipTests clean deploy
         env:
           MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
+          MAVEN_GPG_PASSPHRASE: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
```

Comment:
Mark this file as generated in .gitattributes

Reply:
👍